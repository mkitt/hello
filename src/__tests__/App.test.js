import React from 'react'
import renderer from 'react-test-renderer'
import App from '../App'

describe('/', () => {
  it('shows the correct snapshot tree for the / page', () => {
    const component = (
      <App />
    )
    const tree = renderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
