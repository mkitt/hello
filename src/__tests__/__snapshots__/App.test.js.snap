// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`/ shows the correct snapshot tree for the / page 1`] = `
<h1>
  Hello.
</h1>
`;
